client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	isExpiring,
+	isIncludedWithPlan,
+	isPaidWithCreditCard,
+	isPaidWithCredits,
+	isPaidWithPayPalDirect,
+	paymentLogoType,
+	hasPaymentMethod,
+} from 'calypso/lib/purchases';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import PaymentLogo from 'calypso/components/payment-logo';
+import type { Purchase } from 'calypso/lib/purchases/types';
+
+export default function PaymentInfoBlock( { purchase }: { purchase: Purchase } ): JSX.Element {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+
+	if ( isIncludedWithPlan( purchase ) ) {
+		return (
+			<PaymentInfoBlockWrapper purchase={ purchase }>
+				{ translate( 'Included with plan' ) }
+			</PaymentInfoBlockWrapper>
+		);
+	}
+
+	if ( hasPaymentMethod( purchase ) ) {
+		if ( isPaidWithCredits( purchase ) ) {
+			return (
+				<PaymentInfoBlockWrapper purchase={ purchase }>
+					{ translate( 'Credits' ) }
+				</PaymentInfoBlockWrapper>
+			);
+		}
+
+		const logoType = paymentLogoType( purchase );
+
+		if ( isPaidWithCreditCard( purchase ) ) {
+			return (
+				<PaymentInfoBlockWrapper purchase={ purchase }>
+					<PaymentLogo type={ logoType } disabled={ isExpiring( purchase ) } />
+					{ purchase.payment.creditCard?.number ?? '' }
+				</PaymentInfoBlockWrapper>
+			);
+		}
+
+		if ( isPaidWithPayPalDirect( purchase ) ) {
+			return (
+				<PaymentInfoBlockWrapper purchase={ purchase }>
+					<PaymentLogo type={ logoType } disabled={ isExpiring( purchase ) } />
+					{ translate( 'expiring %(cardExpiry)s', {
+						args: {
+							cardExpiry: moment( purchase.payment.expiryDate, 'MM/YY' ).format( 'MMMM YYYY' ),
+						},
+					} ) }
+				</PaymentInfoBlockWrapper>
+			);
+		}
+
+		return (
+			<PaymentInfoBlockWrapper purchase={ purchase }>
+				<PaymentLogo type={ logoType } disabled={ isExpiring( purchase ) } />
+			</PaymentInfoBlockWrapper>
+		);
+	}
+
+	return (
+		<PaymentInfoBlockWrapper purchase={ purchase }>{ translate( 'None' ) }</PaymentInfoBlockWrapper>
+	);
+}
+
+function PaymentInfoBlockWrapper( {
+	children,
+	purchase,
+}: {
+	children: React.ReactNode;
+	purchase: Purchase;
+} ) {
+	const translate = useTranslate();
+	return (
+		<aside aria-label={ String( translate( 'Payment method' ) ) }>
+			<em className="manage-purchase__detail-label">
+				<span>{ translate( 'Payment method' ) }</span>
+				{ isExpiring( purchase ) && (
+					<div className="manage-purchase__detail-label-subtitle">
```

Comment:
Technically this might be invalid HTML (`<div>` inside `<em>`)?

Reply:
Yeah... you're right. I considered that. What I _wanted_ was for there to be no margin between the title (which should probably be an `h` tag of some level rather than an `em` since it's made a block-level element by the CSS) and the subtitle, but I want the subtitle to also be a block-level element. I had originally done that with CSS, but then I realized that I may as well just use HTML that made it a block-level element. However, this results in the issue you describe. I'll refactor the `em` and the subtitle to hopefully resolve this.